portfolio/src/main/java/com/google/sps/data/AuthenticationInfo.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.data;
+
+/** An object to hold authentication information*/
+public final class AuthenticationInfo {
+
+  private final boolean isLoggedIn;
+  private final String loginUrl;
+  private final String logoutUrl;
+
+  public AuthenticationInfo(boolean isLoggedIn, String url) {
```

Comment:
nit: using one parameter for both login and logout url is error prone.

This can be improved by static create method:
```
private AuthenticationInfo(boolean isLoggedIn, String loggedInUrl, String loggedOutUrl){
...
}

public static AuthenticationInfo createLoggedInInfo(String url){
     return new AuthenticationInfo(true, url, "");
} 
```

Reply:
That makes more sense, thank you!